package.json

```diff
@@ -0,0 +1,252 @@
+        }
+    },
+    "scripts": {
+        "vscode:prepublish": "del-cli dist/template && webpack --mode production && cd ./src/template && node ../../node_modules/copy/bin/cli.js ./**/* ../../dist/template",
+        "vscode:package": "node ./node_modules/vsce/vsce package",
+        "lint": "node node_modules/tslint/bin/tslint **/*.ts -e **/*.d.ts -e node_modules/**/*.ts -p .",
+        "fixlint": "node node_modules/tslint/bin/tslint **/*.ts -e **/*.d.ts -e node_modules/**/*.ts -p . --fix",
+        "compile": "del-cli dist/template && webpack --mode development && node ./node_modules/copy/bin/cli.js ./src/template/**/* dist/template",
+        "preinstall": "npx npm-force-resolutions",
+        "pretest": "npm run test-compile",
+        "test": "node ./out/tests/TestsRunHost.js",
+        "test-compile": "npm run compile && del-cli out/models/recognizers/template && tsc -p ./ && cd ./src/template && node ../../node_modules/copy/bin/cli.js ./**/* ../../out/models/recognizers/template"
+    },
+    "extensionDependencies": [
+        "ms-kubernetes-tools.vscode-kubernetes-tools",
+        "mindaro-dev.file-downloader"
+    ],
+    "devDependencies": {
+        "@types/mocha": "^5.2.7",
+        "@types/node": "^12.12.67",
+        "@types/node-fetch": "^2.5.7",
+        "@types/vscode": "1.57.0",
+        "copy": "^0.3.2",
+        "del-cli": "^4.0.1",
+        "mocha": "^6.2.2",
+        "mocha-trx-reporter": "^3.3.1",
+        "ts-loader": "^6.2.1",
+        "tslint": "^5.20.1",
+        "tslint-eslint-rules": "^5.4.0",
+        "tslint-microsoft-contrib": "^6.2.0",
+        "typemoq": "^2.1.0",
+        "typescript": "^3.5.3",
+        "vsce": "^2.9.2",
+        "vscode-test": "^1.0.0",
+        "webpack": "^4.46.0",
+        "webpack-cli": "^3.3.10"
+    },
+    "optionalDependencies": {
+        "fsevents": "^2.1.3"
+    },
+    "dependencies": {
+        "@microsoft/vscode-file-downloader-api": "^1.0.1",
+        "@types/tmp": "^0.1.0",
+        "chokidar": "^3.5.3",
+        "command-exists": "^1.2.8",
+        "glob-promise": "^3.4.0",
+        "guid-typescript": "^1.0.9",
+        "lodash": "^4.17.21",
+        "node-fetch": "^2.6.7",
+        "opener": "^1.5.1",
+        "portfinder": "^1.0.28",
+        "request": "^2.88.0",
+        "request-promise-native": "^1.0.9",
+        "telaug": "0.0.13",
+        "tmp": "^0.1.0",
+        "username": "^5.1.0",
+        "vscode-extension-telemetry": "^0.1.6",
+        "vscode-kubernetes-tools-api": "^1.3.0",
+        "vscode-tas-client": "^0.1.22",
+        "watchpack": "^2.3.1",
+        "glob": "~7.2.3",
+        "glob-parent": "~5.1.2"
+    },
+    "overrides": {
+        "serialize-javascript": "3.1.0"
+    },
+    "homepage": "https://aka.ms/bridge-to-k8s-vscode-quickstart",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Azure/vscode-bridge-to-kubernetes"
+    },
+    "extensionMetadata": {
+        "expectedCLIVersion": "$Expected_CLI_Version$"
```

Comment:
💡 How this `$Expected_CLI_Version$` gets passed for this meta information?

Reply:
this is passed via the CI/CD build, we are planning to change this when we open source this.

Reply:
Sounds good, this is good for this repo to lock it to latest stable release, thank you. 📓 => version I will update this to is `1.0.20220816.1`.

Reply:
it is 1.0.20220816.2 please refer this blob link to get latest CLI version - https://bridgetokubernetes.azureedge.net/zip/lks.json but this would change once OSS release pipelines are used by vscode changes. we need to change the way vscode download them from github releases instead of blob storage. A new feature for vscode.

Reply:
💡 ok latest reside here - https://bridgetokubernetes.azureedge.net/zip/lks.json verion: `1.0.20220816.2` , I will update this place and then this PR is ready to be merged as it is, because it has old code - just need original authors go-ahead. Thanks.